docs/speech-usage.rst

```diff
@@ -196,10 +196,10 @@ See: `Single Utterance`_
     ...                            sample_rate=16000)
     ...     responses = client.streaming_recognize(sample,
     ...                                            single_utterance=True)
-    ...     alternatives = list(responses)
-    >>> print(alternatives[0].transcript)
+    ...     results = list(responses)
+    >>> print(results[0].alternatives[0].transcript)
     hello
-    >>> print(alternatives[0].confidence)
+    >>> print(results[0].alternatives[0].confidence)
```

Comment:
Nope, this is the surface change I was talking about.

This is an example of the current structure this PR changes vs before.
```diff
+ {
+     is_final: True
+     stability: 0.9234354
+     alternatives. []
+ }
- [
-   {
-      transcript: 'some transcript'
-      confidence: 0.9124354 
-    },
- ]
```